addons/binding/org.openhab.binding.systeminfo/src/main/java/org/openhab/binding/systeminfo/internal/model/SysteminfoInterface.java

```diff
@@ -19,6 +19,11 @@
  */
 public interface SysteminfoInterface {
 
+    /**
+     * Initialize logic for the Systeminfo implementation
```

Comment:
To be sure can you search the current source code of openhab en esh whether we use the English initialised or the American Initialized? @kaikreuzer do you hav an opinion about U.K. English or American English?

Reply:
I found that both the English and The American versions are used

Reply:
@kaikreuzer wdyt about English or American?

Reply:
We usually tend to use American english as our developer/user base in the US is much bigger than in the UK.